hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestRecordLevelIndex.scala

```diff
@@ -418,48 +419,6 @@ class TestRecordLevelIndex extends RecordLevelIndexTestBase {
     validateDataAndRecordIndices(hudiOpts)
   }
 
-  @ParameterizedTest
-  @CsvSource(value = Array(
-    "COPY_ON_WRITE,6", "COPY_ON_WRITE,8", "MERGE_ON_READ,6", "MERGE_ON_READ,8"
-  ))
-  def testRLIWithDTClustering(tableType: String, tableVersion: Int): Unit = {
```

Comment:
why deleted this test?

Reply:
This was an unintended change. Added it back.